net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml

```diff
@@ -77,6 +77,12 @@
         <type>checkbox</type>
         <help>This will skip chain and host verification.</help>
     </field>
+    <field>
+        <id>output.influx_v2_timeout</id>
+        <label>Timeout</label>
+        <type>text</type>
+        <help>Flush timeout for the v2 Telegraf output, formatted as a string. If not provided, will default to 5s. 0s means no timeout (not recommended). Increase if you get timeout errors.</help>
```

Comment:
Can you remove the "s" from integer values here, as it would suggest to also insert "s" in the field.